prep-html-script-tag/index.html

```diff
@@ -0,0 +1,15 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>HTML Script Tag</title>
+  </head>
+  <body>
+    <script>
+      console.log('I have added JS to a script tag in the HTML!');
+    </script>
+    <script src="main.js"></script>
+    <script>
+      console.log('I have added a script file via the src attribute!');
+    </script>
```

Comment:
After you make the above change ^, you can delete this `<script>` tag. We only need the 2